src/GridLines.tsx

```diff
@@ -222,3 +221,91 @@ const WeekStripes = ({ monthStart, timeScale }: WeekStripesProps) => {
 
   return <g>{lines}</g>
 }
+
+/* ·················································································································· */
+/*  Hour
+/* ·················································································································· */
+
+interface HourLineProps {
+  xPosition: number
+  height?: string
+}
+
+const HourLine = ({ xPosition, height }: HourLineProps) => {
+  const xAxisTheme = useTimelineTheme().xAxis
+  const classes = useMonthViewStyles(xAxisTheme)
+  return (
+    <line
+      className={classes.line}
+      x1={xPosition}
+      y1={0}
+      x2={xPosition}
+      y2={height ? height : '80%'}
+      strokeWidth={1}
+    />
+  )
+}
+
+const TEN_SECOND_OFFSET_MS = 10000;
```

Comment:
Suggestions welcome on this value. If it's too big then it runs the risk of overlapping at the closest zoom level. However, if its too small then it won't be far away from the edges. Maybe another idea is to align the text on the left to the right side and vice versa for the other side.